tardis/montecarlo/src/test_cmontecarlo.c

```diff
@@ -274,7 +278,7 @@ bool
 test_montecarlo_bound_free_scatter(){
 	double DISTANCE = 1e13;
 	montecarlo_bound_free_scatter(rp, sm, DISTANCE);
-	return rpacket_get_status(rp);
+	return (rpacket_get_status(rp)!=0);
```

Comment:
@mreineck no mark it as xfail in the python file. Here's how: https://github.com/tardis-sn/tardis/blob/master/tardis/montecarlo/tests/test_cmontecarlo.py#L74 add `@pytest.mark.xfail` above that. that will show us continually that this is an xfail.